include/oneapi/dpl/pstl/pstl_config.h

```diff
@@ -102,14 +102,6 @@
     (_MSC_VER >= 1912 && _MSVC_LANG >= 201703L) ||                                                                     \
         (_GLIBCXX_RELEASE >= 9 && __GLIBCXX__ >= 20190503 && __cplusplus >= 201703L)
 
-#define _PSTL_CPP14_2RANGE_MISMATCH_EQUAL_PRESENT                                                                      \
```

Comment:
The pstl_config file is here only to simplify comparison and sync-up with the upstream configuration (because of different file names); IIRC we keep it as-is with no changes introduced. If we want to clean up this file, we should really do it in the PSTL upstream, not here.

Reply:
Will investigate in separated task.